web/rainmaker/dev-packages/egov-ui-framework-core/src/ui-containers/DynamicMdmsContainer/index.js

```diff
@@ -175,6 +175,7 @@ class DynamicMdmsContainer extends Component {
             fullwidth: true,
             isClearable: true,
             required: isRequired,
+            required: requiredValue,
```

Comment:
check this line once, different values get assigned to the same variable
           required: isRequired,
           required: requiredValue,